### PR TITLE
PR for issue #178

### PR DESCRIPTION
**Pull request by kansology**
Remaining_billing_cycles unavailable in subscription element #178

### CODE REVIEW

File: src/main/java/com/ning/billing/recurly/model/Subscription.java [hunk 1]
```diff
@@ -243,6 +246,14 @@ public DateTime getCurrentPeriodStartedAt() {
         return currentPeriodStartedAt;
     }
 
+    public Integer getRemainingBillingCycles() {
+        return remainingBillingCycles;
+    }
+
+    public void setRemainingBillingCycles(Integer remainingBillingCycles) {
```
Comment (pierre) [inline src/main/java/com/ning/billing/recurly/model/Subscription.java]: This should probably be an `Object` instead to handle `nil` (see `integerOrNull`).

Comment (bhelx) [inline src/main/java/com/ning/billing/recurly/model/Subscription.java]: We should probably mark these `final` as well.

**Comment by kansology** [comment]
Changes incorporated, please review.

File: src/main/java/com/ning/billing/recurly/model/Subscription.java [hunk 2]
```diff
@@ -461,6 +473,9 @@ public boolean equals(final Object o) {
         if (expiresAt != null ? expiresAt.compareTo(that.expiresAt) != 0 : that.expiresAt != null) {
             return false;
         }
+        if (remainingBillingCycles != null ? remainingBillingCycles.compareTo(that.remainingBillingCycles) != 0 : that.remainingBillingCycles != null) {
```
Comment (bhelx) [inline src/main/java/com/ning/billing/recurly/model/Subscription.java]: I think this should be changed from 

```java
remainingBillingCycles.compareTo(that.remainingBillingCycles) != 0
```

to 


```java
!remainingBillingCycles.equals(that.remainingBillingCycles)
```

**Comment by kansology** [comment]
Changes incorporated, please review.

**Comment by bhelx** [comment]
Thanks a bunch @kansology!

**Pull request merged**